controller/test/integration/workerpool_integration.sh

```
@@ -0,0 +1,319 @@
+POD_NAME=""
+for i in {1..90}; do
+  POD_NAME=$(kubectl -n ${NAMESPACE} get pods -o jsonpath='{range .items[*]}{.metadata.name}{" "}{.spec.containers[0].image}{"\n"}{end}' | awk '$2 ~ /alpine/ {print $1; exit}')
+  if [ -n "${POD_NAME}" ]; then
+    echo "Found task pod: ${POD_NAME}"
+    break
+  fi
+  sleep 2
+done
+
+if [ -n "${POD_NAME}" ]; then
+  kubectl -n ${NAMESPACE} wait --for=condition=Succeeded pod/${POD_NAME} --timeout=120s || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)